lib/cluster.js

```diff
@@ -291,14 +291,12 @@ function masterInit() {
       var match = execArgv[i].match(/^(--debug|--debug-(brk|port))(=\d+)?$/);
 
       if (match) {
+        let debugPort = process.debugPort + debugPortOffset;
+        ++debugPortOffset;
```

Comment:
That's not really the point of this PR. There have been a few other PRs that have attempted to add checking, and they are still open.